src/main/java/com/github/pedrovgs/androidwifiadb/action/AndroidWiFiADBAction.java

```diff
@@ -65,13 +65,26 @@ public void run() {
         NotificationType.INFORMATION);
   }
 
-  @Override public void showErrorConnectingDeviceNotification(Device device) {
+    @Override
+    public void showDisconnectedDeviceNotification(Device device) {
+        showNotification(ANDROID_WIFI_ADB_TITLE, "Device '" + device.getName() + "' disconnected.",
+                NotificationType.INFORMATION);
+    }
+
+    @Override public void showErrorConnectingDeviceNotification(Device device) {
     showNotification(ANDROID_WIFI_ADB_TITLE,
         "Unable to connect device '" + device.getName() + "'. Review your WiFi connection.",
         NotificationType.INFORMATION);
   }
 
-  @Override public void showADBNotInstalledNotification() {
+    @Override
+    public void showErrorDisconnectingDeviceNotification(Device device) {
+        showNotification(ANDROID_WIFI_ADB_TITLE,
+                "Unable to disconnect device '" + device.getName() + "'. Review your WiFi connection.",
+                NotificationType.INFORMATION);
+    }
+
+    @Override public void showADBNotInstalledNotification() {
     showNotification(ANDROID_WIFI_ADB_TITLE,
         "Android SDK not found. Please, review your project configuration and ensure that you are working on an "
                 + "Android project.", NotificationType.ERROR);
```

Comment:
I don't see any change in this file related to code. If I'm not wrong, the only thing you've changed here is the ``@Override`` annotation declaration to be in top of the method signature instead of inline. This project uses Square's code style and checkstyle configuration. Can you revert these changes, install the Square code style you can find here: https://github.com/square/java-code-styles and reformat the code again? :)